weave-js/src/components/Switch/index.tsx

```diff
@@ -2,15 +2,18 @@ import * as Switch from '@radix-ui/react-switch';
 import React from 'react';
 import {twMerge} from 'tailwind-merge';
 
+export type SwitchSize = 'small' | 'normal';
```

Comment:
Feels inconsistent for button sizes to be small/medium/large but switch sizes to be small/normal.